src/domains/games/router.js

```diff
@@ -9,6 +9,7 @@ const gameController = require('./controller');
 
 const routes = {
   'GET: /': [authentication, authorization(['Player', 'Admin']), validation(gameValidator.list), gameController.list],
+  'POST: /view-count/:id': [gameController.addViewCount],
```

Comment:
Mungkin dsini bisa nambah validator mas untuk paramsnya

Reply:
thank you mas udah jeli, aku missed hehe @Nursalli 

```js
gameId: z.string().regex(/^\d+$/).transform(Number)
```

ref:https://github.com/colinhacks/zod/discussions/330

cc @grgsferry

Reply:
done added mas, thankyou!